e2e/vm/additional_disk_test.go

```diff
@@ -23,11 +28,18 @@ var testAdditionalDisk = func(o *option.Option) {
 			oldImagesOutput := command.StdoutStr(o, "images", "--format", "{{.Name}}")
 			gomega.Expect(oldImagesOutput).Should(gomega.ContainSubstring(savedImage))
 
-			command.Run(o, "run", "--name", containerName, savedImage)
+			command.Run(o, "volume", "create", userVolume)
+			command.Run(o, "run", "--name", containerName, "-v", fmt.Sprintf("%s:/myvolume", userVolume),
+				savedImage, "sh", "-c", fmt.Sprintf("touch /myvolume/%s; ls /myvolume", userFile))
+
+			ginkgo.DeferCleanup(command.Run, o, "volume", "rm", userVolume)
```

Comment:
nit: defer clean up the thing right after it's created

```suggestion
			command.Run(o, "volume", "create", userVolume)
			ginkgo.DeferCleanup(command.Run, o, "volume", "rm", userVolume)

			command.Run(o, "run", "--name", containerName, "-v", fmt.Sprintf("%s:/myvolume", userVolume),
				savedImage, "sh", "-c", fmt.Sprintf("touch /myvolume/%s; ls /myvolume", userFile))
```

Reply:
Updated.